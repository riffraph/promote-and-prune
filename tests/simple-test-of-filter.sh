#!/bin/sh

TESTDATAFOLDER=./simpleTestOfFilterDataset

# simple test of filter

mkdir -p $TESTDATAFOLDER/folder1

head -c 8 /dev/urandom > $TESTDATAFOLDER/folder1/file1
head -c 1024 /dev/urandom > $TESTDATAFOLDER/folder1/file2


identify.sh $TESTDATAFOLDER
promote.sh $TESTDATAFOLDER


# rm -rf $TESTDATAFOLDER
