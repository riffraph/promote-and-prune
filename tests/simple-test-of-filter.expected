-- IDENTIFY --
8 ./simpleTestOfFilterDataset/folder1/file2
-- PROMOTE --
TODO
